Lib9c/Delegation/Delegatee.cs

```diff
@@ -464,5 +515,66 @@ private void TransferRemainders(LumpSumRewardsRecord record)
                 }
             }
         }
+
+        private void MigrateLumpSumRewardsRecords()
+        {
+            List<LumpSumRewardsRecord> records = new();
+            if (!(Repository.GetCurrentLumpSumRewardsRecord(this) is LumpSumRewardsRecord record))
+            {
+                return;
+            }
+
+            while (record.LastStartHeight is long lastStartHeight)
+            {
+                records.Add(record);
+                record = Repository.GetLumpSumRewardsRecord(this, lastStartHeight)
+                    ?? throw new InvalidOperationException(
+                            $"Lump sum rewards record for #{lastStartHeight} is missing");
+            }
+
+            records.Reverse();
+
+            RewardBase? rewardBase = null;
+            RewardBase? newRewardBase = null;
+            foreach (var recordEach in records)
+            {
```

Comment:
List 의 Reverse 는 항목을 재배치 하기 때문에 Reverse 를 사용하지 않고 for 문으로 뒤에서 순회를 하는게 좋을 것 같습니다.

```suggestion
            RewardBase? rewardBase = null;
            RewardBase? newRewardBase = null;
            for (var i = records.Count - 1; i >= 0; i--)
            {
                var records = records[i];
```